provider/kubernetes/kubernetes.go

```diff
@@ -219,8 +219,14 @@ func (p *Provider) loadIngresses(k8sClient Client) (*types.Configuration, error)
 					}
 				}
 
-				if rule := getRuleForPath(pa, i); rule != "" {
-					templateObjects.Frontends[baseName].Routes[pa.Path] = types.Route{
+				rule, err := getRuleForPath(pa, i)
+				if err != nil {
+					log.Errorf("fail to get rule from ingress path: %s", err)
```

Comment:
Please update the log message as following:

```go
log.Errorf("Failed to get rule for ingress %s/%s: %s", i.Namespace, i.Name, err)
```
(grammar/style of the first word + inclusion of the Ingress object's namespace and name)